src/components/PaginationRow/PaginationRow.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MenuItem, DropdownButton } from '../../index';
+
+const defaultMessages = {
```

Comment:
👍 . I like this addition of making the message text a parameter. Will also add this in #143

Reply:
this is a must for i18n... so thanks again for this. I'll continue to be on the lookout for these kind of things.